modules/virt-creating-custom-resources-hpp.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-custom-resources-hpp_{context}"]
+= Creating host path provisioner custom resources
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, the system will:
+
+* Update the legacy HPP
+* Create the CSI HPP based on the contents of the legacy HPP custom resource.
+
+If you install {VirtProductName} version 4.10 on a new cluster, the system does not perform these tasks. Instead, you must:
+
+* Create the CSI driver custom resource.
```

Comment:
This reads to me like the user should create some CSI related custom resource. I think it might be better to say something like `Create the HPP CR using the storage pool format` format might not be the best word either. But saying `CSI driver custom resource` kind of implies you are creating something that is part of the CSI spec, but it is more about the storage pool than anything.